nginx/nginx.conf.d/coala.io.conf

```diff
@@ -1,77 +1,70 @@
-    location /reviewsprint { return 302 https://docs.google.com/forms/d/e/1FAIpQLSd4vHafTyY4RW--fOyIVecBM0WKNEeF-RyFvUn83jCF9ou2tg/viewform; }
-    location /reply     { return 302 https://github.com/coala/coala/wiki/Reply-Templates; }
-    location /linespots { return 302 https://gitlab.com/sims1253/Linespots; }
-    location /usability { return 302 https://docs.google.com/forms/d/e/1FAIpQLSe9lZxuYEKlvxXzQUOTwrre3CQMNsks7eOzEl49_2q5vlDl0w/viewform; }
-    location /starwars  { return 302 https://www.youtube.com/watch?v=JWVCMjKU_10; }
-    location /docs      { return 302 https://docs.coala.io/; }
-    location /api       { return 302 https://api.coala.io/; }
-    location /viperform { return 302 https://docs.google.com/forms/d/e/1FAIpQLSdtdIF5CLnO2erAc41yLRoEvUMXyt3ZWUOVJ5LSqpwZEYF03A/viewform; }
-    location /romania   { return 302 https://www.bigmarker.com/remote-meetup/Open-Source-and-Google-Summer-of-Code-8-Gorgeous-Short-Presentations; }
-    location /thesis    { return 302 https://github.com/coala/coala/wiki/Writing-a-Thesis-with-coala; }
-    location /install   { return 302 http://docs.coala.io/en/latest/Users/Install.html; }
-    location /comania   { return 302 https://github.com/coala/coala/wiki/Hackathon-comania; }
-    location /pizza     { return 302 https://docs.google.com/forms/d/e/1FAIpQLSdt_zc_bgAE6vqTypSBnWk240AEnhCE7NBmpU1nLnFBuaivqA/viewform; }
-    location /help      { return 302 http://api.coala.io/en/latest/Developers/coala_settings.html; }
-    location /devsetup  { return 302 https://api.coala.io/en/latest/Developers/Development_Setup.html; }
-    location /community_issues { return 302 https://github.com/pulls?q=is:open+user:coala+label:"community_team"+sort:created-asc; }
-    location /fosdem2017 { return 302 http://eepurl.com/cABthT; }
-    location /gitmate    { return 302 https://coala.io/#/gitmate; }
-    location /nginx.txt { return 302 coala.io.conf; }
-    location /links     { return 302 coala.io.conf; }
-    location /csoc      { return 302 https://docs.google.com/forms/d/e/1FAIpQLSeR8WKkZA1R0gBDjJqfeI96jgHe9mt8zmAVm1YtB5FpJFD9gQ/viewform?usp=sf_link; }
-    location /recipes   { return 302 https://github.com/coala/recipes; }
-
-    location /autocontrib { return 302 https://docs.google.com/presentation/d/1fYx1aiSSj_w1vw017tBYfSGbfgl38sMWg3qlhvimYu8/edit?usp=sharing; }
-    location /exterminate { return 302 https://github.com/adtac/exterminate/; }
-    location /exterminatep { return 302 https://docs.google.com/presentation/d/1qJpXMgA8_WJNO3eUV_ueNiU8w_1fTethNvEuU2XHnWs/edit?usp=sharing; }
-
-    location /feedback { return 302 https://goo.gl/DcTPbn; }
-
-    location /coala.io.conf {
-      add_header Content-Type text/plain;
-      root       /etc/nginx/conf.d;
+    location /links {
+        root /usr/share/nginx/html;
     }
+
+    location /newcomer { return 301 https://api.coala.io/en/latest/Developers/Newcomers_Guide.html; }
```

Comment:
improve the template so that it adds whitespace to the tab ` { ` so that your diff doesnt have any changes. (or at least very few changes)